docs/guides/auth.md

```diff
@@ -180,7 +180,7 @@ var client = new Greeter.GreeterClient(channel);
 ####Base case - No encryption/authentication
```

Comment:
In what sense? Obviously, yes, they aren't actually an example of authenticating with anything but is it not useful to see the difference between "no authentication" and "here's how you add x form of authentication"? Or is it the case that they shouldn't be under Authenticating with Google as they're also not authenticating with anything else?

Perhaps we need to reorganize the whole examples section by language, so then you'd have eg a Ruby section with first "no authentication", then "SSL/TLS authentication", then "authenticating with Google"....